File: src/main/java/ru/c47harsis/Main.java
package ru.c47harsis;

import lombok.extern.slf4j.Slf4j;
import org.telegram.telegrambots.meta.TelegramBotsApi;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;

@Slf4j
public class Main {

    public static void main(String[] args) {
        FleaMarketBot fleaMarketBot = new FleaMarketBot();
        log.info("Starting {}", fleaMarketBot.getBotUsername());
        try {
            TelegramBotsApi botsApi = new TelegramBotsApi(DefaultBotSession.class);
            botsApi.registerBot(fleaMarketBot);
        } catch (TelegramApiException e) {
            log.error(e.getMessage());
        }
        log.info("{} successfully started", fleaMarketBot.getBotUsername());
    }
}

File: src/main/kotlin/ru/c47harsis/Main.kt
package ru.c47harsis

import mu.KotlinLogging
import org.telegram.telegrambots.meta.TelegramBotsApi
import org.telegram.telegrambots.meta.exceptions.TelegramApiException
import org.telegram.telegrambots.updatesreceivers.DefaultBotSession

private val log = KotlinLogging.logger {}

fun main() {
    val fleaMarketBot = FleaMarketBot()
    log.info { "Starting ${fleaMarketBot.botUsername}" }
    try {
        val botsApi = TelegramBotsApi(DefaultBotSession::class.java)
        botsApi.registerBot(fleaMarketBot)
    } catch (e: TelegramApiException) {
        log.error { "An error occurred when starting ${fleaMarketBot.botUsername}, message: ${e.message}" }
    }
    log.info { "${fleaMarketBot.botUsername} successfully started" }
}